### DISCUSSION TITLE
Newbe question about repositories

### DISCUSSION THREAD

**Discussion by jorisvandermeulen** [community/community#23191] [New to GitHub]
<p>I’ve read about it and I somehow still don’t get it. So … here it goes.</p>
<p>In VSCode I have a workspace which contains 3 folders, but only 1 gets noted by Source Control which makes me wonder if that is due the way I set it up.</p>
<p>I now think that<br>
a github repository equals a specific directory in my workspace (and it’s subdirectories).</p>
<p>And is it correct that state that:<br>
if I want the other directories to be ‘synced’ to github I should create 2 more repositories or move those folders into the other (already set up) folder?</p>
<p>Trying to get the whole picture here.</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="3Pinter" data-post="1" data-topic="157407">
<div class="title">
<div class="quote-controls"></div>
 3Pinter:</div>
<blockquote>
<p>if I want the other directories to be ‘synced’ to github I should create 2 more repositories or move those folders into the other (already set up) folder?</p>
</blockquote>
</aside>
<p>That depends on if (or how) the code in those directories is connected. It’s generally good practice to keep code for one project in one repository.</p>
<p>Note that Git doesn’t <em>sync</em> anything. You need to explicitly commit and push the things you want to be on Github.</p>
